google-cloud-storage/src/test/scala/akka/stream/alpakka/googlecloud/storage/SessionSpec.scala

```diff
@@ -16,11 +16,11 @@ class SessionSpec extends WordSpec {
     "report the path of the missing service account file" in {
       implicit val as: ActorSystem = null
       implicit val mat: Materializer = null
-      val session = new Session(GoogleAuthConfiguration("missingpath"), Seq.empty)
+      val session = new Session(GoogleAuthConfiguration("/missing/path/test.json"), Seq.empty)
```

Comment:
I think this break the build on non unixy machines. Use `TempDirectory.location()` and then combine with a more specific file name that will not exist.

Reply:
Switching to Paths.get(System.getProperty("java.io.tmpdir")).resolve("idonotexist.json") as TempDirectory is not accessible